package.json

```diff
@@ -0,0 +1,56 @@
+{
+    "name": "gpii-devpmt",
+    "productName": "devpmt",
+    "version": "1.0.0",
+    "description": "Developers Preference Testing Tool",
+    "main": "standalone.js",
+    "scripts": {
+        "start": "electron standalone.js",
+        "watch-sass": "gulp",
+        "build-sass": "gulp sass"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/sgithens/gpii-devpmt"
+    },
+    "author": "Steven Githens <sgithens@raisingthefloor.org>",
+    "license": "BSD-3-Clause",
+    "bugs": {
+        "url": "https://issues.gpii.net/"
+    },
+    "homepage": "https://github.com/sgithens/gpii-devpmt",
+    "dependencies": {
+        "electron": "1.4.1",
+        "foundation-sites": "6.3.0",
+        "gpii-express": "1.0.2",
+        "gpii-handlebars": "the-t-in-rtf/gpii-handlebars#GPII-2474",
+        "gulp-autoprefixer": "^3.1.0",
+        "handlebars-helpers": "0.8.0",
+        "infusion": "3.0.0-dev.20170127T130413Z.103de6e",
```

Comment:
This version is outdated by about a year. A reasonable choice would be either the infusion bound to by the version of universal that you end up binding to, or else the one referenced by your version of gpii-handlebars (when updated)

Reply:
Upgraded to most recent dev version.